lib/spack/spack/test/conftest.py

```diff
@@ -844,7 +844,8 @@ def mutable_database(database_mutable_config, _store_dir_and_cache):
     # Restore the initial state by copying the content of the cache back into
     # the store and making the database read-only
     store_path.remove(rec=1)
-    store_cache.copy(store_path, mode=True, stat=True)
+    copy_tree(str(store_cache), str(store_path))
+    # store_cache.copy(store_path, mode=True, stat=True)
```

Comment:
This commented-out line should be removed

Reply:
Thought I caught this, my bad. Will remove now.